services/app-api/forms/mcpar.json

```diff
@@ -1100,16 +1092,16 @@
                 "type": "textarea",
                 "validation": "text",
                 "props": {
-                  "label": "C1.IV.1 State's definition of \"critical incident,\" as used for reporting purposes in its MLTSS program",
-                  "hint": "If this report is being completed for a managed care program that covers LTSS, what is the definition that the state uses for \"critical incidents\" within the managed care program? Respond with \"N/A\" if the managed care program does not cover LTSS."
+                  "label": "C1.IV.1 State's definition of “critical incident,” as used for reporting purposes in its MLTSS program",
```

Comment:
label text: imo the comma should be outside the quotes for `critical incident` but they were also inside in the original copy. if that's what design wants that's fine